File: Modulo-01-C/03_mario/mario.c
#include <cs50.h>
#include <stdio.h>


int main(void)
{
	int x;
	do
	{
		x = get_int("Numero positivo: ");
	}

	while (x < 1 || x > 8);
	{
		for(int i = 0; i < x; i++)
    	{
			for (int f = (x-2); f >= i; f--)
			{
					printf(" ");
			}
			for (int j = 0; j <= i; j++)
			{
					printf("#");
			}
            for (int j = 0; j <= 1; j++)
			{
					printf(" ");
			}
            for (int j = 0; j <= i; j++)
			{
					printf("#");
			}
			printf("\n");
    	}
	}
}

File: Modulo-06-Python/22_mario_more/mario.py
from cs50 import get_int

# Pede entrada do usuário ate que receba uma válida
height = 0
while (height > 8 or height < 1):
    height = get_int("Height:\n")

# Imprime uma pirâmide de hashes
for i in range(1, height +1):
    print(" " * (height -i), end="")
    print("#" * i, end="  ")
    print("#" * i)
